lib/elixir/lib/system.ex

```diff
@@ -114,6 +114,11 @@ defmodule System do
     end
   end
 
+  # Returns OTP version that Elixir was compiled with.
+  defmacrop get_otp_version do
```

Comment:
just to be clear I was proposing to rename this `defmacrop get_otp_version` to `defmacrop compiled_otp_release`